### DISCUSSION TITLE
adding a component as part of the list

### DISCUSSION THREAD

**Discussion by paulwvnjohi** [mskocik/svelecte#60] [Ideas]
question: currently would it be possible to add a component in the list as such 
<img width="158" alt="Screen Shot 2021-11-23 at 10 04 30 AM" src="https://user-images.githubusercontent.com/30245020/142982999-7a1c1392-30e0-44cc-9964-e54fcbcebd9e.png">
if not can i open a PR with the change sometime?

**Comment by mskocik** [comment]
Can you elaborate a little more about this one? How should it work?

Sidenote: you know there is a `creatable` property, which enables you to create new items, Or is not enough for your needs?

**Reply by paulwvnjohi** [reply]
actually that serves my use case. Thanks. looking  forward to contributing.

**Reply by mskocik** [reply]
Great to hear, because adding additional component would not be possible in current design anyway 😃

**Reply by mskocik** [reply]
In `v4.0` it is possible to add own markup with slot `list-header`. Check docs for more info.
